### PR TITLE
Validations are ready to fly  🚀 after your review 

### PR DESCRIPTION
**Pull request by serkan-ozel-2015400123**
* **Input Validations of:**
* Sign in
* Sign up
* Event create is implemented 

+ Minor improvement : Price currency selector is implemented.
and a little bit of styling is done.

Any other ideas are welcome. Please review this :)

### CODE REVIEW

**Comment by serkan-ozel-2015400123** [comment]
can you review this after merging the cors pr ?

**Comment by okyksl** [comment]
@serkan-ozel-2015400123 sorry for the late reply. I though we talked to move common sass into the **global sass** file. I think there are couple of things to do here:

* Report any error case with a small text under the field.
* Report any error returned from the **backend** with a popover.

**Comment by serkan-ozel-2015400123** [comment]
This is done 
> Report any error returned from the backend with a popover.

**Comment by serkan-ozel-2015400123** [comment]
> Report any error case with a small text under the field.

This is also done to auth only

**Comment by serkan-ozel-2015400123** [comment]
Part of this, I could not test, due to the bug I mentioned before, i.e in local I am not able to sign in. 
Can you please test if you can and merge this too @okyksl ? Thanks

File: frontend/src/global.scss [hunk 1]
```diff
@@ -9,4 +9,13 @@
 @import "~@ionic/angular/css/text-alignment.css";
 @import "~@ionic/angular/css/text-transformation.css";
 @import "~@ionic/angular/css/flex-utils.css";
+.ng-valid:not(form)  {
```
Comment (okyksl) [inline frontend/src/global.scss]: It seems like this **red line** conflicts with ionic blue line.

File: frontend/src/global.scss [hunk 2]
```diff
@@ -9,4 +9,13 @@
...
+  border-bottom:2px solid #42A948; /* green */
+}
 
+.ng-invalid:not(form)  {
+  border-bottom:2px solid Tomato /* red */
```
Comment (okyksl) [inline frontend/src/global.scss]: It seems like this **red line** conflicts with ionic **blue line**.

File: frontend/src/global.scss [hunk 3]
```diff
@@ -9,4 +9,13 @@
...
+}
+.notify{
+  font-size: 0.7em;
```
Comment (okyksl) [inline frontend/src/global.scss]: We can increase the font-size a bit, it feels too small for me.

File: frontend/src/app/events/pages/event/event.page.ts [hunk 4]
```diff
@@ -54,5 +56,19 @@ export class EventPage implements OnInit, OnDestroy{
...
+        header: 'Wait..',
+        subHeader: 'The event couldn\'t be loaded',
+        message: 'We cannot get the event information: ' + errMessage,
+        buttons: ['Close']
+      });
+    await alert.present();
+  }
+  private handleError(error: HttpErrorResponse) {
+
+    this.presentAlert(`${JSON.stringify(error.error)}`);
```
Comment (okyksl) [inline frontend/src/app/events/pages/event/event.page.ts]: We need to have a specific **error field** that is agreed with the **backend team**. Is this settled with **backend** team?

File: frontend/src/app/events/pages/event-create/event-create.page.html [hunk 5]
```diff
@@ -18,26 +18,39 @@
       <ion-label> Event Title : </ion-label>
       <ion-input type="text" value="" max="150" formControlName="name" class="inputStyle"></ion-input>
     </ion-item>
+    <ion-label floating class="notify" *ngIf="!this.form.get('name').valid">The event name can include characters and space.</ion-label>
```
Comment (okyksl) [inline frontend/src/app/events/pages/event-create/event-create.page.html]: Let's add **dirty** control as well. We don't want to show errors when the user didn't entered anything. Can you check whether **fields are dirty as well** in `*ngIf` statement.

Comment (serkan-ozel-2015400123) [reply frontend/src/app/events/pages/event-create/event-create.page.html]: Done can you check

**Comment by serkan-ozel-2015400123** [comment]
Changed title from "Ionic App" to "Welcome to Actopus" by the way

**Comment by okyksl** [comment]
@serkan-ozel-2015400123 great work! I resolved the conflict with the branch `auth-module-ui`.

**Pull request merged**